### PR TITLE
[Blazor] Host and Deploy - {TARGET FRAMEWORK} placeholder + naming

### PR DESCRIPTION
**Pull request by hakenr**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/blazor/host-and-deploy/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/6870d8ce9af079be830758aa879126a780d9c8b5/aspnetcore/blazor/host-and-deploy/index.md) | [aspnetcore/blazor/host-and-deploy/index](https://review.learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/index?branch=pr-en-us-34330) |

### CODE REVIEW

File: aspnetcore/blazor/host-and-deploy/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Publish locations:
 :::moniker range=">= aspnetcore-8.0"
 
 * Blazor Web App: The app is published into the `/bin/Release/{TARGET FRAMEWORK}/publish` folder. Deploy the contents of the `publish` folder to the host.
-* Blazor WebAssembly: The app is published into the `bin\Release\net8.0\browser-wasm\publish\` folder. To deploy the app as a static site, copy the contents of the `wwwroot` folder to the static site host.
+* Standalone Blazor WebAssembly: The app is published into the `bin\Release\{TARGET FRAMEWORK}\browser-wasm\publish\` folder. To deploy the app as a static site, copy the contents of the `wwwroot` folder to the static site host.
```
Comment (guardrex) [inline aspnetcore/blazor/host-and-deploy/index.md:51]: We do this to make sure that the placeholder is translated in localized articles ...

```suggestion
* Standalone Blazor WebAssembly: The app is published into the `bin\Release\{TARGET FRAMEWORK}\browser-wasm\publish\` folder, where the `{TARGET FRAMEWORK}` placeholder is the target framework. To deploy the app as a static site, copy the contents of the `wwwroot` folder to the static site host.
```

**Review by guardrex — APPROVED** [review]
# 👍

File: aspnetcore/blazor/host-and-deploy/index.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Publish locations:
 :::moniker range=">= aspnetcore-8.0"
 
 * Blazor Web App: The app is published into the `/bin/Release/{TARGET FRAMEWORK}/publish` folder. Deploy the contents of the `publish` folder to the host.
-* Standalone Blazor WebAssembly: The app is published into the `bin\Release\{TARGET FRAMEWORK}\browser-wasm\publish\` folder. To deploy the app as a static site, copy the contents of the `wwwroot` folder to the static site host.
+* Standalone Blazor WebAssembly: The app is published into the `bin\Release\{TARGET FRAMEWORK}\browser-wasm\publish\` folder, where the `{TARGET FRAMEWORK}` placeholder is the target framework. To deploy the app as a static site, copy the contents of the `wwwroot` folder to the static site host.
```
Comment (hakenr) [inline aspnetcore/blazor/host-and-deploy/index.md:51]: @guardrex Placeholder explanation is covered by line 64 for all the occurrences?

Comment (guardrex) [reply aspnetcore/blazor/host-and-deploy/index.md:51]: ... and I might move the remark above these passages. It's usually better to mention it beforehand.

Comment (guardrex) [reply aspnetcore/blazor/host-and-deploy/index.md:51]: No ... what I'll do is update the next versioned set of lines and remove the Line 64 remark.

**Pull request merged**